README.md

```diff
@@ -172,3 +171,5 @@ Removed following and all unused discs from Azure Cloud.
 Removed following and all unused pips from Azure Cloud.
 ['test-axodawttrw-pip0']
 ```
+
+### [How to run the cloudwash setup locally](https://github.com/RedHatQE/cloudwash/blob/master/Docs/Run_cloudwash_locally.md)
```

Comment:
```suggestion
### [How to run the cloudwash setup locally using Minikube](https://github.com/RedHatQE/cloudwash/blob/master/Docs/Run_cloudwash_locally.md)
```

Reply:
Also in MD file, it is good to mention that this helps in running the cloudwash as minikube/pod container.